doc/man1/openssl-verification-options.pod

```diff
@@ -343,10 +343,19 @@ Set policy variable inhibit-policy-mapping (see RFC5280).
 
 =item B<-purpose> I<purpose>
 
-The intended use for the certificate. If this option is not specified, this
-command will not consider certificate purpose during chain verification.
-Currently accepted uses are B<sslclient>, B<sslserver>, B<nssslserver>,
-B<smimesign>, B<smimeencrypt>.
+The intended use for the certificate.
+Currently defined purposes are C<sslclient>, C<sslserver>, C<nssslserver>,
+C<smimesign>, C<smimeencrypt>, C<crlsign>, C<ocsphelper>, C<timestampsign>,
+and C<any>.
+If peer certificate verification is enabled, by default the TLS implementation
+as well as the commands B<s_client> and B<s_server> check for consistency
+with TLS server or TLS client use, respectively.
+
+Note that (as of 2021) there is no consensus even at standardization level
```

Comment:
This paragraph gives the wrong impression.  I'd argue that it should just be removed.  5280 is completely clear about the meaning of those bits.  The fact that other organizations do their own thing doesn't mean that 5280 is wrong.  (I find it ironic you point to consensus when you were adamant about 5280's pre-emininence in other PR's)

If you want to keep this paragraph, then I suggest it be changed:
```
While IETF RFC 5280 says that B<d-kp-serverAuth> and B<id-kp-clientAuth> are only for WWW use, in practice they are used for all TLS clients and servers, and this is what OpenSSL assumes as well.
```

Reply:
Note that there is an errata about this: https://www.rfc-editor.org/errata/eid5802

Reply:
> 5280 is completely clear about the meaning of those bits. The fact that other organizations do their own thing doesn't mean that 5280 is wrong. (I find it ironic you point to consensus when you were adamant about 5280's pre-emininence in other PR's)

In this detail I find the 5280 shortsigthted/anachronistic and would agree to the other organizations. At least the IETF should have clarified since long whether they really mean that just WWW is the right thing there, and if so, they should have provided alternative EKUs for other TLS uses.

I like the sentence you proposed and took it over (with two small improvements).

Reply:
@kroeckx as discussed elsewhere, the errata is just filed, it's just an opinion at this point this was discussed at https://github.com/openssl/openssl/issues/13965#issuecomment-778254473.

By the way, the LAMPS working group just removed "update 5280" from its recharting proposal.

Reply:
Interesting that "update 5280" is no more planned yet.
Is this postponed or is no need seen for that?
Has there been a discussion on handling the open errata?

Reply:
Since 5280 isn't being modified, the errata continues to just sit there as there is no way to handle it.
As for why 5280 isn't being re-opened, the mail archive might be useful https://mailarchive.ietf.org/arch/browse/spasm/   Ryan Sleevi's messages are particularly cogent; TLDR - there is no one single PKI.